View/AppStartPoint.cpp

```diff
@@ -0,0 +1,73 @@
+#include "ViewInitializer.h"
+#include "StatisticsDisplayer.h"
+#include <string>
+#include <iostream>
+#include <GL/glut.h>
+#include <GL/glew.h>
+using namespace MeshIS::View;
+
+vector<function<void()>> renderFunctions;
```

Comment:
I made some changes, but OpenGL requires Display and Reshape function to be static, so used variables have to be static too.